README.md

```diff
@@ -35,7 +35,7 @@ of the arguments as they might be defined in the application environment `config
 or `runtime.exs` files.
 
 ```elixir
-config :my_app, :elsa,
+config :my_app, :elsa_fi,
```

Comment:
Do we want to change this? Leaving it the same would make it a 1-step-shorter drop-in replacement for others.

Reply:
It's kind of a pain either way unfortunately :(

- Elixir gets suspicious and throws up warnings if it sees you putting in configurations that aren't for the app name that you configured.
- I needed a unique app name so that Hex could publish it as its own thing (at least without additional hackery).
- Apparently barely anyone is using this anyway or it would probably get updated, so I don't think we'll cause a lot of churn by renaming the app.

I ended up deciding it was probably the lesser of two evils to make everything be "elsa_fi".